openapi/definitions.yaml

```diff
@@ -102,6 +102,8 @@ NewMessage:
       $ref: "#/MessageContent"
     default_addresses:
       $ref: "#/NewMessageDefaultAddresses"
+    required_channel_privacy_level:
```

Comment:
since we have only two privacy levels, why don't we start with a simple boolean?

Reply:
@francescopersico let's discuss about this

Reply:
We decided to set the privacy at Service level. I already have a local branch with a field called "requireSecureChannel" defined at Service level (option boolean with default of FALSE). I am going to close this PR and open a new one that implements the behaviour decided on PivotalTracker.